StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1001CommasMustBeSpacedCorrectly.cs

```diff
@@ -109,13 +109,13 @@ private static void HandleCommaToken(SyntaxTreeAnalysisContext context, SyntaxTo
             if (missingFollowingSpace)
             {
                 // comma must{} be {followed} by a space
-                context.ReportDiagnostic(Diagnostic.Create(Descriptor, token.GetLocation(), string.Empty, "followed"));
+                context.ReportDiagnostic(Diagnostic.Create(Descriptor, token.GetLocation(), OpenCloseSpacingCodeFixProvider.InsertFollowing, string.Empty, "followed"));
             }
 
             if (hasPrecedingSpace)
```

Comment:
:exclamation: Can you reverse the order here, so problems with preceding trivia are reported before problems with trailing trivia?